packages/app/src/cli/models/extensions/specification.ts

```diff
@@ -144,10 +152,19 @@ export function createConfigExtensionSpecification<TConfiguration extends BaseCo
     appModuleFeatures,
     transform: resolveAppConfigTransform(spec.transformConfig),
     reverseTransform: resolveReverseAppConfigTransform(spec.schema, spec.transformConfig),
+    simplify: resolveSimplifyAppConfig(spec.simplify),
     experience: 'configuration',
   })
 }
 
+function resolveSimplifyAppConfig(simplifyConfig?: SimplifyConfig) {
```

Comment:
adds new simplify method

Reply:
It looks nice!

Just one doubt: if I understand it correctly, we are iterating through the config modules, merging them to the final config and applying the simplification in that moment. And the purpose is to merge two different modules (webhooks and compliance webhooks). But what happens if the simplify method, which belongs to the webhooks module, is called before the compliance webhooks module is processed? I'm afraid it could be working by chance.

The easiest solution I can think of would be:
- Process all the modules as before, merging them in a single config
- Iterate a second time through all the modules and call the simplify method if present. That way, we ensure the config already contains all the modules.

Does it make sense?

Reply:
Ah yes, I think Alvaro's comment below will solve for this - I think I misunderstood where to put this code when it was mentioned that we should iterate over all the specifications! Thank you for the clarification 😄